src/net.cpp

```diff
@@ -383,7 +385,7 @@ CNode* FindNode(const NodeId nodeid)
     return NULL;
 }
 
-CNode* ConnectNode(CAddress addrConnect, const char *pszDest, bool fCountFailure)
+CNode* ConnectNode(CAddress addrConnect, const char *pszDest, bool fCountFailure, bool fFeeler)
```

Comment:
I think this is unnecessary and confusing, indeed. We shouldn't pass around state just for nitty logging purposes.